proto/src/determined/command/v1/command.proto

```diff
@@ -42,4 +42,6 @@ message Command {
   string exit_status = 12;
   // The associated job id.
   string job_id = 13;
+  // Timestamp of last update to the user object.
+  int64 user_version = 14;
```

Comment:
I'd put it next to `username` field even though the tag number is not sequential.